template.yaml

```diff
@@ -26,14 +45,14 @@ Resources:
     Type: AWS::Serverless::Function 
     Properties:
       CodeUri: getSignedURL/
-      Handler: app.handler
-      Runtime: nodejs16.x
+      Handler: index.handler
+      Runtime: nodejs18.x
       Timeout: 3
```

Comment:
updating the node engine is probably good, but renaming the handler is too broad